pkg/configurator/methods.go

```diff
@@ -36,6 +36,10 @@ func (c *Client) GetMeshConfig() configv1alpha2.MeshConfig {
 	return c.getMeshConfig()
 }
 
+func (c *Client) GetMeshRootCertificates() ([]configv1alpha2.MeshRootCertificate, error) {
```

Comment:
It looks like this is never used do we need it?

Reply:
Yeah it's not anymore; this is a relic before we came up with the watch approach